config/204-psp.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: extensions/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: knative-serving-psp
+  labels:
+    serving.knative.dev/release: devel
+    serving.knative.dev/controller: "true"
+spec:
+  allowPrivilegeEscalation: true
+  allowedCapabilities:
+  - 'NET_ADMIN'
```

Comment:
We only need this for cases where we install istio with sidecar injection enabled, right?

Reply:
Yes, serving activator and autoscaler need "NET_ADMIN" to startup.

Reply:
I think I'd rather have this in `test/config` until we figure out if/what we want to ship.

Reply:
@yu2003w I don't think it's the actual activator/autoscaler but rather the istio sidecars (or init containers in this case) that need these permissions.

Reply:
Sorry, I misunderstood that. It's Istio sidecar needs the permission.

Reply:
@mattmoor Moved psp.yaml to test/config.
When both psp and Istio sidecar automatically injection are enabled, deployments in namespace serving-tests also need 'NET_ADMIN'. So I also defined role and rolebinding within namespace serving-tests.